### PR TITLE
Add unsetService function

### PR DESCRIPTION
**Pull request by pradel**
Add unsetService function

### CODE REVIEW

File: __tests__/index.ts [hunk 1]
```diff
@@ -551,6 +551,23 @@ describe('Mongo', () => {
     });
   });
 
+  describe('unsetService', () => {
+    it('should not convert id', async () => {
```
Comment (TimMikeladze) [inline __tests__/index.ts]: Where's the `expect` in this `it` block?

Comment (pradel) [reply __tests__/index.ts]: there is no expect because it should throw is the test is failing

Comment (davidyaha) [reply __tests__/index.ts]: Shouldn't be expect.toThrow?

File: src/mongo.ts [hunk 2]
```diff
@@ -348,6 +348,26 @@ export class Mongo implements DBInterface {
...
   }
 
+  public async unsetService(
+    userId: string,
+    serviceName: string
+  ): Promise<void> {
+    const id = this.options.convertUserIdToMongoObjectId
+      ? toMongoID(userId)
+      : userId;
+    await this.collection.update(
```
Comment (davidyaha) [inline src/mongo.ts]: use `updateOne`?

File: __tests__/index.ts (hunk 1 above)
Comment (pradel) [reply __tests__/index.ts]: It should not throw, it will throw if the function was converting the id

File: src/mongo.ts (hunk 2 above)
Comment (pradel) [reply src/mongo.ts]: Would be better but actually all code use update so for consistency maybe let's keep this change for another pr?

Comment (davidyaha) [reply src/mongo.ts]: sure

File: __tests__/index.ts (hunk 1 above)
Comment (davidyaha) [reply __tests__/index.ts]: not sure I'm getting the point then.. sorry

Comment (pradel) [reply __tests__/index.ts]: So the function check that if you have the option convertUserIdToMongoObjectId if you pass a non valid mongo string it should not throw. If the option was not handled properly the function should throw.
Do you think about a better way to test it?

Comment (davidyaha) [reply __tests__/index.ts]: mmm either mock the update func and expect the same user id you passed will be the best option I think

Comment (davidyaha) [reply __tests__/index.ts]: Thanks for explaining the issue though. I missed that on first read..

Comment (pradel) [reply __tests__/index.ts]: Fixed :)

**Pull request merged**